rb/lib/selenium/webdriver/remote/bridge.rb

```diff
@@ -541,6 +541,11 @@ def element_attribute(element, name)
           execute :getElementAttribute, id: element.ref, name: name
         end
 
+        # Backwards compatibility for w3c
+        def element_property(element, name)
+          execute :getElementAttribute, id: element, name: name
```

Comment:
this should try use `getElementProperty` and if the driver doesnt understand that then fallback to `getElementAttribute` (not sure if this is already handled elsewhere)